packages/design-system-react-native/src/components/Text/README.md

```diff
@@ -62,13 +62,13 @@ Available `TextColor` options:
 
 Optional prop to adjust the font weight.
 
-| TYPE                          | REQUIRED | DEFAULT             |
-| :---------------------------- | :------- | :------------------ |
-| [FontWeight](./Text.types.ts) | No       | `FontWeight.Normal` |
+| TYPE                          | REQUIRED | DEFAULT              |
```

Comment:
Found an instance of FontWeight.Normal which will need to be updated

![Screenshot 2025-03-28 at 2 51 17 PM](https://github.com/user-attachments/assets/5662572d-4f70-41f6-b90e-1905780477ea)